upup/pkg/fi/cloud.go

```diff
@@ -44,6 +54,24 @@ type SubnetInfo struct {
 	CIDR string
 }
 
+// CloudInstanceGroup is the cloud backing of InstanceGroup.
+type CloudGroup struct {
```

Comment:
So these 2 classes are what I'd like to see in a new package, rather than making `fi` bigger...